docs/source/pbft-source-documentation.rst

```diff
@@ -0,0 +1,38 @@
+*************************
+PBFT Source Documentation
```

Comment:
Since PBFT is a binary application (not a library), do you think it still makes sense to use the term "API" here? I'll put it in for now, and change it if anyone comes up with something better - I can't think of anything off the top of my head.